pandas/core/ops/__init__.py

```diff
@@ -59,6 +59,37 @@
     rxor,
 )
 
+# -----------------------------------------------------------------------------
+# constants
+ARITHMETIC_BINOPS: Set[str] = {
```

Comment:
Just as FYI if you create a container with literal values mypy will infer that it is a `Set[str]` for you, so not required to specify. Have to specify if the container is empty on instantiation

Reply:
Good to know!